composer.json

```
@@ -22,10 +22,10 @@
     "php": "^7.2",
     "ext-json": "*",
     "amphp/amp": "^2.1.2",
-    "amphp/artax": "^3.0.14",
+    "amphp/http-client": "^4.0.0@rc",
```

```suggestion
    "amphp/http-client": "^4.0.0",
```

@prolic Commit this before merging when the stable is out.